doc/nrf/app_dev/device_guides/thingy53/building_thingy53.rst

```diff
@@ -68,8 +68,8 @@ For the compatible Wi-Fi samples in the |NCS|, see the :ref:`wifi_samples` secti
 Programming methods for Thingy:53
 *********************************
 
-You can program the firmware on the Nordic Thingy:53 using an external debug probe and a 10-pin JTAG cable, using :ref:`Visual Studio Code <thingy53_build_pgm_vscode>`, :ref:`command line <thingy53_build_pgm_command_line>`, or the `Programmer app <Programming Nordic Thingy:53_>`_ from nRF Connect for Desktop.
-You can also program applications running on both the network and application core using the built-in MCUboot serial recovery mode, using the `Programmer app`_ from nRF Connect for Desktop or `nRF Util <Programming application firmware using MCUboot serial recovery_>`_.
+You can program the firmware on the Nordic Thingy:53 using an external debug probe and a 10-pin JTAG cable, using :ref:`Visual Studio Code <thingy53_build_pgm_vscode>`, :ref:`command line <thingy53_build_pgm_command_line>`, or the `Programmer app <prog_thingy_53>`_ from nRF Connect for Desktop.
+You can also program applications running on both the network and application core using the built-in MCUboot serial recovery mode, using the `Programmer app <nRF Connect Programmer>`_ from nRF Connect for Desktop or `nRF Util <Programming application firmware using MCUboot serial recovery_>`_.
```

Comment:
These are editing the hyperlinks already defined in links.txt (https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/links.txt#L673 and https://github.com/nrfconnect/sdk-nrf/blob/main/doc/nrf/links.txt#L670). Not sure why these edits are required.

Reply:
fixed